### PR TITLE
Adding support for displaying messages regarding rise-storage events

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

**Comment by stulees** [comment]
@alex-deaconu Please review, thanks!

File: src/widget/video-folder.js [hunk 1]
```diff
@@ -59,6 +91,13 @@ RiseVision.VideoFolder = (function (gadgets) {
   function pause() {
     var frameObj = _frameController.getFrameObject(_currentFrame);
 
+    _viewerPaused = true;
+
+    if (_noFilesFlag) {
+      _startNoFilesTimer();
+      return;
+    }
```
Comment (alex-deaconu) [inline src/widget/video-folder.js]: @stulees curious, why are you starting the timer on `pause`? Thanks.

**Comment by alex-deaconu** [comment]
@stulees one comment re timer. It seems that the Widget would send done when it is paused too. 

Otherwise, PR looks good. Thanks.

**Comment by stulees** [comment]
@alex-deaconu Great catch. It wasn't causing any issue, but definitely meant to clear the timer, not start it. Last commit correct this, thanks!

**Comment by alex-deaconu** [comment]
Awesome, thanks for fixing. :+1:

**Comment by stulees** [comment]
@donnapep Please review last commit, thanks

**Comment by donnapep** [comment]
:+1:

**Pull request merged**